[BSv5] Refactor Click-to-copy (#1447)
google/docsy: CHANGELOG.md

**New**:

- **Click to copy button for Chroma-highlighted code blocks**: If you already

>   implemented this functionality on your website, you can disable it. More
>   information on the [Chroma highlighting docs][chroma-docsy].

Reviewer: ```suggestion
  implemented this functionality on your website, you can disable it. For details,
  see [Code highlighting with Chroma][chroma-docsy].
```

Author: Updated

> **Breaking changesdocs

Reviewer: Copy-paste error?

```suggestion
**Breaking changes**:
```

Author: More like a random vim keypress :(  Fixed.
